stdlib/set.mli

```diff
@@ -118,6 +118,17 @@ module type S =
        The elements of [s] are presented to [f] in increasing order
        with respect to the ordering over the type of the elements. *)
 
+    val map: (elt -> elt) -> t -> t
+    (** [map f s] creates a new set with elements [f a0],
```

Comment:
I would replace "creates a new set with elements" by "is a set with elements" (describe the value denoted by the expression).